--- a/node_modules/yjs-orderedtree/src/ytree.js
+++ b/node_modules/yjs-orderedtree/src/ytree.js
@@ -83,8 +83,9 @@ export class YTree {
     }
     else if (yMap.size === 0) {
       this._ymap = yMap;
-      const root = this._ymap.set("root", new Y.Map());
+      const root = new Y.Map();
       root.set("_parentHistory", new Y.Map());
+      this._ymap.set("root", root);
     }
     else {
       throw new Error(
