diff --git a/node_modules/yjs-orderedtree/src/ytree.js b/node_modules/yjs-orderedtree/src/ytree.js
index 18880ac..2019358 100644
--- a/node_modules/yjs-orderedtree/src/ytree.js
+++ b/node_modules/yjs-orderedtree/src/ytree.js
@@ -127,7 +127,7 @@ export class YTree {
           if (event.path.length == 0) {
             event.changes.keys.forEach((change, key) => {
               if (change.action === "update") {
-                throw new Error("[ytree] The node should not be updated");
+                console.warn("[ytree] Update on node detected and allowed:", key);
               }
             });
             // Recalculate virtual/computed map whenever a node is added or deleted.
